src/intrinsic/mod.rs

```diff
@@ -1086,7 +1086,8 @@ impl<'a, 'gcc, 'tcx> Builder<'a, 'gcc, 'tcx> {
 }
 
 fn try_intrinsic<'gcc, 'tcx>(bx: &mut Builder<'_, 'gcc, 'tcx>, try_func: RValue<'gcc>, data: RValue<'gcc>, _catch_func: RValue<'gcc>, dest: RValue<'gcc>) {
-    if bx.sess().panic_strategy() == PanicStrategy::Abort {
+    if bx.sess().panic_strategy() == PanicStrategy::Abort || true {
+        // TODO(bjorn3): Properly implement unwinding and remove the `|| true` once this is done.
```

Comment:
Is the `|| true` there to make it use this code when unwinding is enabled?
If so, can you specify this in the comment, please, as this is not clear to me?

Reply:
Correct. Will add a comment.

Reply:
Done